src/main/java/org/openstreetmap/atlas/checks/validation/intersections/BuildingRoadIntersectionCheck.java

```diff
@@ -33,25 +36,44 @@
 {
     private static final List<String> FALLBACK_INSTRUCTIONS = Arrays
             .asList("Building (id-{0,number,#}) intersects road (id-{1,number,#})");
+    private static final String INDOOR_KEY = "indoor";
+    private static final String YES_VALUE = "yes";
     private static final long serialVersionUID = 5986017212661374165L;
 
     private static Predicate<Edge> ignoreTags()
     {
         return edge -> !(Validators.isOfType(edge, CoveredTag.class, CoveredTag.YES)
-                || Validators.isOfType(edge, TunnelTag.class, TunnelTag.BUILDING_PASSAGE));
+                || Validators.isOfType(edge, TunnelTag.class, TunnelTag.BUILDING_PASSAGE,
+                        TunnelTag.YES)
+                || Validators.isOfType(edge, AreaTag.class, AreaTag.YES)
+                || (edge.getTag(INDOOR_KEY).isPresent() && edge.tag(INDOOR_KEY).equals(YES_VALUE))
```

Comment:
This condition could be improved with `YES_VALUE.equals(edge.tag(INDOOR_KEY))`. Non-null expected value can be used as reference and that way null values could be handled without extra presence check.